docs/modules/ROOT/pages/plugins.adoc

```diff
@@ -216,25 +215,22 @@ The endpoint accepts a `POST` request. Example post request body:
 
 The parameters are passed directly to your plugin's `handler` function.
 
-== Debugging
+== Responses
 
-When invoking a plugin, the response will include:
+API responses use the `ApiResponse` envelope: `{ success, data, error }`.
 
-- `logs`: The logs from the plugin execution.
-- `return_value`: The returned value of the plugin execution.
-- `error`: An error message if the plugin execution failed.
-- `traces`: A list of messages sent between the plugin and the Relayer instance. This includes all the payloads passed through the `PluginAPI` object.
+- Success (HTTP 200): `data` contains `{ result: <your return>, logs?, traces? }`, `error` is null.
```

Comment:
Would like to get your opinion here.

using result wrapper inside of data seems like to introduce many leveles.

What about using just data for result and moving out logs and traces to top level or to wrap it in top level under metadata or something. Does that make sense?

That way data would contain just result like other endpoints.